platform-hub-api/app/services/announcements_processor_service.rb

```diff
@@ -83,7 +83,7 @@ def email_addresses_for announcement
         []
       end
 
-    contact_lists = [announcement.deliver_to['contact_lists']].compact.flatten
+    contact_lists = Array(announcement.deliver_to['contact_lists'])
```

Comment:
this won't remove potential `nil` entries.

Reply:
I believe it does?

```ruby
[2.3.3p222]> Array(nil)
=> []
```

Reply:
Oh, I think I know what you mean:

```ruby
[2.3.3p222]> Array([nil])
=> [nil]
```